src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -20,42 +20,50 @@
  * @author Fabien Potencier <fabien@symfony.com>
  *
  * @implements \IteratorAggregate<int, \DOMNode>
+ *
+ * @deprecated since Symfony 7.1, use Symfony\Component\DomCrawler\DomCrawler instead
```

Comment:
No trigger deprecations here, just like we did for `ContainerAwareInterface` in #50615 to prevent deprecations to pop with class autoloading when using `DomCrawler`.